frontend/src/Components/OpportunityCard/OpportunityCard.js

```diff
@@ -0,0 +1,35 @@
+import styles from "./OpportunityCard.css";
+
+function OpportunityCard(props) {
+    const text = props.desc;
+    const MAX_LENGTH = 40;
+
+    return (
+        <div className="opportunityCard">
+            <table className="cardContent">
+                <tr>
+                    <th className="cardImage">
+                        <img className="cardIcon" src={props.icon} alt="icon" />
```

Comment:
Could you make this an image of the volunteer event (with a default image if no image provided) and then have the "Interest icons" render as well somewhere on the bottom of the card?